kuzzleio/kuzzle
# Use external vault package

## What does this PR do ?

Enables Kuzzle Core to use the external vault. This solves the problem that Kuzzle was not able to decrypt a secrets file previously encrypted via the CLI (which uses the new, externalized, vault).

### How should this be manually tested?

1. Create a secrets file using the CLI.
2. Start a new Kuzzle instance using this secrets file.
3. Verify that Kuzzle starts up correctly.

### Warning

This feature could not be automatically tested since the integration tests do not allow to setup a specific environment (i.e. using a secrets file with a `KUZZLE_VAULT_KEY` env variable)

### Note

This same bugfix needs to be cherry-picked to Kuzzle V2.
